.circleci/config.yml

```diff
@@ -1428,7 +1818,7 @@ workflows:
           name: nightly_binary_linux_wheel_py3.7_cu113_smoke_test_pip
           python_version: '3.7'
           requires:
-          - nightly_binary_linux_wheel_py3.7_cu113_upload
```

Comment:
I think this is not correct. This change should not affect nightly tests.  The nightly_binary_linux_wheel_py3.7_cu113_smoke_test_pip should require nightly_binary_linux_wheel_py3.7_cu113_upload